Fix imports for items no longer in torh._six
Patch by Simon Branford (University of Birmingham)
--- decode/neuralfitter/utils/dataloader_customs.py.orig	2022-03-19 10:08:57.859040000 +0000
+++ decode/neuralfitter/utils/dataloader_customs.py	2022-03-19 10:09:37.520650000 +0000
@@ -1,6 +1,8 @@
 import torch
 import torch.utils.data
-from torch._six import container_abcs, string_classes, int_classes
+from torch._six import string_classes
+import collections.abc as container_abcs
+int_classes = int
 
 import decode.generic
 
